docs/multicluster/user-guide.md

```diff
@@ -33,6 +33,10 @@ through tunnels among clusters. The ClusterNetworkPolicy replication feature is
 supported since Antrea v1.6.0, and Multi-cluster NetworkPolicy rules are
 supported since Antrea v1.10.0.
 
+Antrea v1.13 introduced incompatible changes to ClusterSet CRDs. If you plan to
```

Comment:
We should say "Antrea v1.13 promoted the ClusterSet CRD version from `v1alpha1` to `v1alpha2`"

Reply:
Updated.